### ISSUE TITLE
toString for method references

### ISSUE THREAD

**Issue by jasontatton** [Concurnas/Concurnas#14]
Currently method references output a toString representation as follows (when running `conc` in in REPL mode):

```
conc> plus = def(a int, b int) => a + b
plus ==> repl$$repl$$$Lambda0repl0@5e2028a3
```
A better toString representation would be:
```
conc> plus = def(a int, b int) => a + b
plus ==> (int, int) int
```

**Comment by jasontatton** [comment]
I am addressing this currently

**Comment by jasontatton** [comment]
resolved in release version 1.14.020 onwards
